app/src/org/commcare/android/database/app/AppDatabaseUpgrader.java

```diff
@@ -40,6 +41,9 @@ public void upgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                 oldVersion = 5;
             }
         }
+        if (oldVersion == 5) {
+            upgradeFiveSix(db);
```

Comment:
@phillipm What do you mean? It's referenced below each time as it falls through the if conditions (they aren't if/elses)